Build/MicroBuildv2.Build.ProjectConfiguration.props

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(SolutionDir)packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props" Condition="Exists('$(SolutionDir)packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')" />
+  <Import Project="$(SolutionDir)packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props"
```

Comment:
Can we group the imports in this file under ImportGroup guarded by the new condition we added? The other conditions need to be still in their individual imports i guess. Btw, should the Microbuild package name be a declared variable so that if the version/name changes we just have one place to update?